tests/Core/Bridge/Symfony/Bundle/DependencyInjection/ConfigurationTest.php

```diff
@@ -195,12 +195,25 @@ private function runDefaultConfigTests(array $doctrineIntegrationsToLoad = ['orm
                     'http_tags' => [
                         'enabled' => true,
                     ],
+                    'purger' => [
+                        'souinBaseHost' => '',
+                        'souinBaseApiPath' => '',
+                        'souinApiSouinPath' => '',
+                        'souinApiAuthenticationPath' => '',
+                        'username' => '',
+                        'password' => '',
```

Comment:
```suggestion
                        'souin' => []
```

do we really need the souinBaseApiPath ? Isn't `host` enough? or maybe a dsn